components/dashboard/ActivityCategoryTable.js

```diff
@@ -0,0 +1,69 @@
+import Box from '@material-ui/core/Box';
+import Divider from '@material-ui/core/Divider';
+import PropTypes from 'prop-types';
+import React from 'react';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import TableBody from '@material-ui/core/TableBody';
+import Table from '@material-ui/core/Table';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Gauge from '../Gauge';
+
+export default function ActivityCategoryTable(props) {
+  const { confidenceByCategories, confidentActivitiesCount, totalActivitiesCount } = props;
+
+  return (
+    <Box mt={2}>
+      <Paper>
+        <Box pt={4} pb={4}>
+          <Gauge
+            percentage={
+              totalActivitiesCount === 0 ? 0 : confidentActivitiesCount / totalActivitiesCount
```

Comment:
I wonder if this should check if the confidentActivitiesCount is less than the totalActivitiesCount?

Reply:
Since we have confidentActivityLogEntries and allActivityLogEntries fully loaded, so the length of the prev will never exceed the next.  Also if you divide 0 by 0 it will return NaN.